server/controllers/userAuth.js

```diff
@@ -448,6 +448,24 @@ const deleteAccount = async (req, resp) => {
   return resp.send();
 };
 
+const updateEmailNotificationSettings = async (req, resp) => {
+  const { _id } = req.user;
+  const { notificationFrequency } = req.body;
+
+  try {
+    await User.findOneAndUpdate(
+      { _id },
+      {
+        emailNotificationsFrequency: notificationFrequency
+      }
+    ).exec();
+
+    resp.send();
+  } catch {
+    resp.send(400);
```

Comment:
Please log the error to ease the debugging.

Reply:
we decided to not log the errors. There is a task in Jira for coding the error logging mechanism